### PR TITLE
Update Edit Command to include format for Staff

### PR DESCRIPTION
**Pull request by rgbpokka**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -109,7 +109,9 @@ Example:
 Edit a guest or staff’s contact details by their passport number. Only edits the fields that have been passed in as
```
Comment (NicolasCwy) [inline docs/UserGuide.md:109]: I think the you could amend this part to include staff id

File: docs/UserGuide.md [hunk 2]
```diff
@@ -197,7 +199,7 @@ _Details coming soon ..._
 Action | Format, Examples
 --------|------------------
 **Add** | `add n/NAME pn/PASSPORT_NUMBER [p/PHONE_NUMBER] [r/ROOM_NUMBER]` <br> e.g., `add n/Bing Cheng pn/T0134568D p/99999999 r/69`
-**Edit** | `edit pn/<PASSPORT_NUMBER> <FIELD_NAME>/<NEW_FIELD_DETAILS>` <br> e.g., `edit pn/X12345678A p/99999999`
+**Edit** | Guest: `edit pn/<PASSPORT_NUMBER> <FIELD_NAME>/<NEW_FIELD_DETAILS>`<br>Staff:`edit pn/<PASSPORT_NUMBER> <FIELD_NAME>/<NEW_FIELD_DETAILS>`<br>e.g.<br>`edit pn/X12345678A p/99999999`<br>`edit sid/S12345678A p/99999999`
```
Comment (NicolasCwy) [inline docs/UserGuide.md:202]: Can you double check that there is a line break after the `e.g` doesn't seem to reflect on my screen

File: docs/UserGuide.md [hunk 3]
```diff
@@ -213,6 +215,7 @@ Parameter | Prefix | Constraints, Examples
 ----------|--------|-----------------------
 **NAME** | `n/` | Blank inputs are not allowed, and should only contain alphanumeric characters. <br> e.g., `n/Bing Cheng`
 **PASSPORT_NUMBER** | `pn/` | Blank inputs are not allowed <br> e.g., `pn/X12345678A`
+**STAFF_ID** | `sid/` | Blank inputs are not allowed <br> e.g., `sid/S12345678A`
```
Comment (NicolasCwy) [inline docs/UserGuide.md:218]: Thanks for adding this!

**Review by NicolasCwy — CHANGES_REQUESTED** [review]
Some small changes to be made!

File: docs/UserGuide.md (hunk 1 above)
Comment (rgbpokka) [reply docs/UserGuide.md:109]: Rectified!

File: docs/UserGuide.md (hunk 2 above)
Comment (rgbpokka) [reply docs/UserGuide.md:202]: Double checked, I did include a line break after the e.g

**Review by NicolasCwy — APPROVED** [review]
LGTM

**Review by calvintanwj — APPROVED** [review]
LGTM

**Pull request merged**